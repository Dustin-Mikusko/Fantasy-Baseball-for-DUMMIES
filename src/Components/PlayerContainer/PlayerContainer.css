.search-player-container {
  margin-top: 200px;
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
}
