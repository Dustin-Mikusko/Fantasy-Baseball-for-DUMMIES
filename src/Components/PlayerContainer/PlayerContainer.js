import React from 'react';
import './PlayerContainer';
import PlayerPage from '../PlayerPage/PlayerPage';

export const PlayerContainer = ({ players }) => {
  const displayPlayers = players.map(player => {
    return (
      <PlayerPage 
        player={player}
      />
    )
  });

  return (
    <div>
      {displayPlayers}
    </div>
  )
}

export default PlayerContainer;
