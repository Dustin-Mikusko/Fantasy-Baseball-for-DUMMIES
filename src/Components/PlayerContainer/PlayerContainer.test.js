import React from 'react';
import { PlayerContainer } from './PlayerContainer';
import { shallow } from 'enzyme';

describe('PlayerContainer', () => {
  it('should match the snapshot with correct data', () => {
    const mockPlayers = [
      {
        name: 'Jose Altuve',
        team: 'Houston Astros',
        position: '2B',
        inches: '6',
        feet: '5',
        weight: '165',
        bats: 'R',
        throws: 'R',
        id: '234'
      }
    ];
    const wrapper = shallow(<PlayerContainer 
      players={mockPlayers}
    />);

    expect(wrapper).toMatchSnapshot();
  }
  )
})
