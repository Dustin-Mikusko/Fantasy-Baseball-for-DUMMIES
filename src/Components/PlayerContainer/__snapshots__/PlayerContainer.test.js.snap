// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerContainer should match the snapshot with correct data 1`] = `ShallowWrapper {}`;
